website/docs/tutorials/structured_config/4_defaults.md

```diff
@@ -68,14 +68,37 @@ db:
   ...
 ```
 
-#### A Note about composition order
-When Hydra composes a config, by default - any values defined in the config are overriding the values coming from configs specified in the Defaults List.
-If you specify a value, Hydra 1.1 will warn that you need to specify `_self_` in the Defaults List to indicate the desired composition order (The above example does not specify a value so there is no warning).
-In cases similar to this one, you will typically want to insert `_self_` as the first item in the Defaults List. 
+### A Note about composition order
+The default composition order in Hydra is that values defined in a config are merged into values introduced from configs in the Defaults List - or in other words - overriding them.
+This behavior can be unintuitive when your primary config is a Structured Config, like in the example above.
+For example, if the primary config is:
+```python {6}
+@dataclass
+class Config:
+    defaults: List[Any] = field(default_factory=lambda:  [
+        "debug/activate",
+        # If you do not specify _self_, it will be appended to the end of the defaults list by default.
+        "_self_"
+    ])
 
+    debug: bool = False
+```
+And `debug/activate.yaml` is overriding the `debug` flag to `True`, the composition order would be such that debug ends up being `False`.  
+To get the desired behavior, explicitly specify `_self_` before `debug/activate.yaml`:
```

Comment:
```suggestion
To change the composition order, explicitly specify `_self_` before `debug/activate.yaml`:
```

Reply:
The sentence does not make a lot of sense.
There are many ways to change the composition order but the suggestion is very specific.

Reply:
You are right.   I'm just not sure if we should say `desired behavior` here though.

Reply:
How about 

> To get the intuitively expected behavior

Reply:
People starting from Hydra 1.1 will have a different expectation of the order. how about
> To have the defaults lists override the subsequent config value,

Reply:
> To get `debug/activate.yaml` to override this config, explicitly specify `_self_` before `debug/activate.yaml`: